sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java

```diff
@@ -71,7 +71,7 @@ public void cancelScheduledMessages(Iterable<Long> sequenceNumbers) {
      * @return A {@link ServiceBusMessageBatch} that can fit as many messages as the transport allows.
      */
     public ServiceBusMessageBatch createBatch() {
```

Comment:
This method name should also be updated to `createMessageBatch()`.

Reply:
Updated the sync ones